opapi/src/generators/errors.ts

```diff
@@ -51,6 +51,10 @@ abstract class BaseApiError<Code extends ErrorCode, Type extends string, Descrip
     }
   }
 
+  toString() {
+    return \`[\${this.type}] \${this.message} (Error ID: \${this.id})\`
```

Comment:
How will this be used? Are going to just interpolate the error in a string in some places?

e.g.
```ts
logger.error(`This is the error: ${error}`)
```

Reply:
In the bot's logs we always see the error with a stack that isn't useful. This way they can just display the error using a default format.
<img width="593" alt="image" src="https://github.com/botpress/packages/assets/10071388/c240706d-2d95-4997-80a6-fee5d6f85494">

Reply:
I think that stack is actually useful? I have used it a few times when investigating issues on a bot, by just retrieving the lambda's customer_code.js and locating the part of the code that triggering it by the line number and character position indicated in the stack.